tests/unit/CorrectNameOptions.spec.ts

```diff
@@ -6,23 +6,25 @@ import CorrectNameOptions from '@/components/common/YourCompany/CompanyName/Corr
 import CorrectCompanyName from '@/components/common/YourCompany/CompanyName/CorrectCompanyName.vue'
 import CorrectNameRequest from '@/components/common/YourCompany/CompanyName/CorrectNameRequest.vue'
 import CorrectNameToNumber from '@/components/common/YourCompany/CompanyName/CorrectNameToNumber.vue'
+import { createPinia, setActivePinia } from 'pinia'
+import { useStore } from '@/store/store'
 
 Vue.use(Vuetify)
 
 const vuetify = new Vuetify({})
+setActivePinia(createPinia())
+const store = useStore()
```

Comment:
I believe this component doesn't use the store, so this can be deleted. Please and thanks!